iotfunctions/data_quality.py

```diff
@@ -0,0 +1,184 @@
+#  Licensed Materials - Property of IBM 
+#  5737-M66, 5900-AAA, 5900-A0N, 5725-S86, 5737-I75
+#  (C) Copyright IBM Corp. 2020 All Rights Reserved.
+#  US Government Users Restricted Rights - Use, duplication, or disclosure
+#  restricted by GSA ADP Schedule Contract with IBM Corp.
+from abc import ABC
+
+import logging
+from iotfunctions.base import (BaseComplexAggregator)
+from iotfunctions.ui import (UISingleItem,
+                             UIMulti)
+import math
+import pandas as pd
+import numpy as np
+from statsmodels.tsa.stattools import (kpss, adfuller, acf, q_stat)
+from statsmodels.stats.diagnostic import acorr_ljungbox
+
+logger = logging.getLogger(__name__)
+
+PACKAGE_URL = 'git+https://github.com/ibm-watson-iot/functions.git@'
+_IS_PREINSTALLED = True
+
+
+class DataQualityChecks(BaseComplexAggregator):
+    """
+    Data Quality module will help asses the quality of incoming sensor data, using the provided metrics.
```

Comment:
I'm not a native speaker but I think a fourth 's' at the tail of 'asses' prevents misunderstandings :-)

Reply:
fyi, 'asses' occurs more often in the description... :-)